26-dracula-ant/technical_specification.md

```diff
@@ -55,38 +57,59 @@ Deletion of work packages should be logged or implemented as deactivation (immed
 
 The work package access token is a random string generated by the Work Package Service on a POST request. Its SHA256 hash is stored in the database. The token itself is not stored, but it is encrypted with the `public_key` and then returned in the response.
 
-The frontend should display the work package ID together with the encrypted work package access token separated by a colon as a single string that can be copied and pasted to the CLI client, so that the CLI client gets both pieces of information.
+The frontend should display the work package ID together with the encrypted work package access token separated by a colon as a single string that can be copied and pasted to the CLI client (GHGA connector), so that the CLI client gets both pieces of information.
 
 ### Work order token
 
 This is a JWT with a very short time to live (maximum 30 seconds). It is signed by the Work Package Service and then encrypted with the `public_key` of the user, similar to the work package access token. It contains the following claims:
 
 - `type`: `download` or `upload`
 - `file_id`: the ID of the file that shall be downloaded or uploaded
+- `file_ext`: the file extension of the requested filed including the dot
```

Comment:
I think this should rather be a part of the work package. The information is available from the beginning on. It does not change between work orders.

Reply:
Just saw that it already is part of it. Can be deleted from the work order token.

Reply:
I thought the DCS might need it to deliver the proper content type by that should be handled by S3 already. So yes, I removed from the work order token.